app/javascript/components/App.js

```diff
@@ -31,23 +31,51 @@ class App extends Component{
 
 
   render () {
+    const {
+      logged_in,
+      sign_in_route,
+      sign_out_route,
+      new_user_route,
+      current_user,
+    } = this.props
     return (
       <>
-      <Header />
       <Router>
+        {
+        !( window.location.pathname === '/') && 
```

Comment:
```suggestion
        window.location.pathname !== '/' && 
```
might be easier to understand